releases.md

```diff
@@ -0,0 +1,201 @@
+# Talos II support in coreboot
+
+## Key changes
+
+1. Added RAM initialization
+2. Added SCOM registers support
```

Comment:
Swap those, SCOM was first. Maybe also mention support for reading data from VPD partition